### ISSUE TITLE
Migrate "Policies" Page

### ISSUE THREAD

**Issue by loganfranken** [UCSantaBarbara/webguide#406]
#384 

**Legacy Page**
https://webguide.ucsb.edu/policies/

**Location in New Hierarchy**
???

Everything on this page still seems relevant; not sure if we want to keep it all on one page still (or if we want to try and situate it closer to its related content)

-----------

Your web pages must comply with UC and UCSB policies. Some good policy
resources to start with are the
[UC Electronics Communications Policy](http://www.ucop.edu/information-technology-services/policies/electronic-communications.html)
and the [UC and UCSB Policies](http://www.policy.ucsb.edu/) pages. In the
recommendations below, WSG provides guidelines and resources to address only
the most common policy issues of concern to UCSB web developers/managers.

### Copyright

The Regents of the University of California hold the copyright to all UC
system domain website content. We recommend you include copyright information
in the [footer](/footer/) of every page on your website.

### Terms of Use

The Office of the Vice Chancellor, Administrative Services, has developed and
maintains a [Terms of Use](http://www.policy.ucsb.edu/terms-of-use/)
document that is specifically concerned with websites in the ucsb.edu domain.
This document defines the ucsb.edu web space and includes sections on Links to
Third Party Sites, Privacy, Access and Use of Services, Restrictions On Use Of
Materials, and the Digital Millennium Copyright Act, as well as disclaimer and
limit of liability statements. The WSG recommends that you include a link on
your web page to the Terms of Use document in your [footer](/footer/).

### Electronic Information Security

If your website collects data, follow guidelines in the
[UC Business and Finance Bulletin IS-3 on Electronic Information Security (PDF)](http://policy.ucop.edu/doc/7000543/).
This policy provides critical information about measures to promote the
security of and access to electronic data.

### Advertising on UCSB Websites

The Office of the Vice Chancellor, Administrative Services, has also developed
and maintains resources related to acknowledgements and advertising on UCSB
websites. These include the following:

[Guidelines for Acknowledgements and Advertising on University Electronic Resources (PDF)](http://www.policy.ucsb.edu/files/docs/policies/advertising-guide.pdf):
A set of guidelines designed to provide basic principles and examples of
acceptable and unacceptable acknowledgements and advertising on UCSB electronic
resources to assist web administrators, designers, and others to abide by
University policy and applicable law.

### Unauthorized Use of Data/Online Applications

The WSG recommends that the following statement be included on the login page
of any UCSB online applications that give access to user data:

"This is official University of California, Santa Barbara information.
Unauthorized access, disclosure, or use of this information may result in
disciplinary action and/or criminal prosecution. See
[Rules of Conduct for Rules of Conduct for University Employees Involved with Information Regarding Individuals](http://www.ucop.edu/ethics-compliance-audit-services/_files/compliance/privacy/rules-of-conduct.pdf)."

For information, read [Civil Code Section 1798.55–1798.57](https://leginfo.legislature.ca.gov/faces/codes_displayText.xhtml?lawCode=CIV&division=3.&title=1.8.&part=4.&chapter=1.&article=10).

### Privacy Notification

UCSB websites that collect personal information from visitors must contain a
formal privacy notification statement, and websites that collect general
information are also encouraged to do so. For details of the guidelines
governing privacy notification, including a definition of "personal
information," see the UCSB
[Advisory on Privacy Notification (PDF)](http://www.policy.ucsb.edu/files/docs/policies/privacy-statement-advisory.pdf).

The Office of the Vice Chancellor, Administrative Services, has developed and
maintains a [Privacy Notification Statement](http://www.policy.ucsb.edu/privacy-notification/)
that summarizes privacy practices for all authorized UCSB websites, including
information collection and use, visitor choices, and security practices. The
WSG recommends that all UCSB websites that collect information from visitors
link to this [Privacy Notification Statement](http://www.policy.ucsb.edu/privacy-notification/).

* **Sites collecting personal information** must display the statement (or a link to the statement) prominently on the page on which the information is requested, and it should be accessible to the individual to read prior to volunteering information about him/herself. The icon or link directing users to the policy should contain the word "privacy."
* **Sites collecting only general information** may include a link in the [footer](/footer/) of the site home page. The link should contain the word "privacy."

### Use of Images and Photo Releases

To use an individual's personal image, photo, name, voice, or creative work on
a UCSB website for the purpose of promoting a UCSB department, program, or
event, obtain the individual's written permission prior to publishing it.

Refer to the [UCSB Risk Management website](http://www.ehs.ucsb.edu/riskmanagement/releases)
for more information.

#### Privacy Notification

When you request permission to use images on your website, we also recommend
that you inform the individual of your privacy practices by including a link
to, or a copy of, your
[Privacy Notification statement](#privacy-notification).

#### Using Images of Public UCSB Events

If you plan to capture images of identifiable individuals during an event to
promote a UCSB department, program, or event, provide those attending with
advance notice by including a statement in your promotional materials
(brochures, websites, et al.) such as, "This event will be photographed or
recorded."

For further details on the use of recording at public UCSB events, please see
[Guidelines for the Use of Individuals' Images and Creative Work (PDF)](http://www.policy.ucsb.edu/files/docs/policies/use-of-images.pdf).

### Additional Policies

For a more comprehensive list of UCSB and UC Policies pertaining to websites,
visit
[UCSB Policies and Procedures - Privacy & Security](http://www.policy.ucsb.edu/information-stewardship/information-practices/privacy-security)

### Linking to UCOP Policies

UC wide Presidential Policies make use of a special permanent URL schema to
avoid broken links. The permanent URLs end in a 7 digit number. The URL schema
is described in the
[UCOP Policy Toolkit](https://www.ucop.edu/ethics-compliance-audit-services/policy/policy-development/policy-toolkit.html).

To summarize the schema: the Sexual Harassment Policy is at:
[http://policy.ucop.edu/doc/4000385/SHSV](http://policy.ucop.edu/doc/4000385/SHSV).
The value after the 7 digit number is generated by the UCOP system and is a
side-effect of UCOP's document versioning. The URL you would use to link this
policy is: [http://policy.ucop.edu/doc/4000385/](http://policy.ucop.edu/doc/4000385/).

**Comment by loganfranken** [comment]
I just went ahead and migrated this to a dedicated page:
https://dev-webguide-ucsb-edu-v01.pantheonsite.io/policies

**Comment by tenken** [comment]
* thanks Logan! for migrating this page.
